internal/v1/api.yaml

```diff
@@ -646,7 +646,28 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/HTTPErrorList'
-
+  /experimental/recommendations:
+    post:
+      summary: List recommended packages.
+      description: "Returns a list of recommended packages for given list of packages."
+      operationId: recommendPackage
+      tags:
+        - recommendations
+      requestBody:
+        content:
+          application/json:
+            schema:
+              "$ref": "#/components/schemas/UploadPackageRequest"
+        required: true
+      responses:
+        '200':
+          description: Return the recommended packages.
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/RecommendationsResponse"
```

Comment:
The original API is POST for some reason, in general this should be GET unless one modifies data or initiates some kind of big change (a job or something). Opinions? @ondrejbudai or @croissanne ?

Reply:
Considering the request has a body, I think a POST is fine.

GET requests with bodies sometimes have weird results when they get cached etc… No strong opinion here but honestly as soon as a body is present it should not be a get imo

Reply:
I will keep it "POST" for now